app/services/DepartureP5MessageService.scala

```diff
@@ -43,9 +43,9 @@ class DepartureP5MessageService @Inject() (
               messagesForMovement.messages.find(_.messageType == DepartureNotification) match {
                 case Some(ie015) =>
                   for {
-                    lrn <- departureMovementP5Connector.getLRN(ie015.bodyPath).map(_.referenceNumber)
-                    ie056  = messagesForMovement.messages.find(_.messageType == RejectedByOfficeOfDeparture)
-                    xPaths = ie056.map(_.functionalErrors.map(_.errorPointer))
+                    lrn   <- departureMovementP5Connector.getLRN(ie015.bodyPath).map(_.referenceNumber)
```

Comment:
On a side note we can probably remove this once this ticket is done, maybe add a TODO so we dont forget:
https://jira.tools.tax.service.gov.uk/browse/CTCP-2924